config/application.rb

```diff
@@ -24,5 +24,7 @@ class Application < Rails::Application
     config.action_controller.action_on_unpermitted_parameters = :raise
     config.active_record.raise_in_transactional_callbacks = true
     config.active_job.queue_adapter = :delayed_job
+    config.i18n.load_path +=
+      Dir[Rails.root.join('config', 'locales', '**/*', '*.{rb,yml}')]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.